wp-autoupdates.php

```diff
@@ -1004,3 +1004,172 @@ function wp_autoupdates_get_update_message() {
+
+
+/**
+ * Render autoupdate column’s content.
+ */
+function wp_autoupdates_add_themes_autoupdates_column_content( $column_name, $stylesheet, $theme ) {
+	if ( ! current_user_can( 'update_themes' ) || ! wp_autoupdates_is_themes_auto_update_enabled() ) {
+		return;
+	}
+	if ( 'autoupdates_column' !== $column_name ) {
+		return;
+	}
+	$themes         = wp_get_themes();
+	$themes_updates = get_site_transient( 'update_themes' );
+	$page           = isset( $_GET['paged'] ) && ! empty( $_GET['paged'] ) ? wp_unslash( esc_html( $_GET['paged'] ) ) : '';
+	$theme_status   = isset( $_GET['theme_status'] ) && ! empty( $_GET['theme_status'] ) ? wp_unslash( esc_html( $_GET['theme_status'] ) ) : '';
+	if ( wp_autoupdates_is_themes_auto_update_enabled() ) {
+		if ( ! isset( $themes[ $stylesheet ] ) ) {
+			return;
+		}
+		$wp_auto_update_themes = get_site_option( 'wp_auto_update_themes', array() );
+		if ( in_array( $stylesheet, $wp_auto_update_themes, true ) ) {
+			$aria_label = esc_attr(
+				sprintf(
+					/* translators: Theme name. */
+					_x( 'Disable automatic updates for %s', 'theme', 'wp-autoupdates' ),
+					esc_html( $themes[ $stylesheet ]->get( 'Name' ) )
+				)
+			);
+			echo '<p>';
+			echo '<span class="theme-autoupdate-enabled">' . __( 'Auto-updates enabled', 'wp-autoupdates' ) . '</span>';
+			echo '<br />';
+
+			$update_message = wp_autoupdates_get_update_message();
+			if ( isset( $themes_updates->response[ $stylesheet ] ) ) {
+				echo '<span class="theme-autoupdate-time">';
+				echo $update_message;
+				echo '<br />';
+				echo '</span>';
+			}
+			if ( current_user_can( 'update_themes', $stylesheet ) ) {
+				echo sprintf(
+					'<a href="%s" class="theme-autoupdate-disable" aria-label="%s">%s</a>',
+					wp_nonce_url( 'themes.php?action=autoupdate&amp;theme=' . urlencode( $stylesheet ) . '&amp;paged=' . $page . '&amp;theme_status=' . $theme_status, 'autoupdate-theme_' . $stylesheet ),
+					$aria_label,
+					__( 'Disable', 'wp-autoupdates' )
+				);
+			}
+			echo '</p>';
+		} else {
+			if ( current_user_can( 'update_themes', $stylesheet ) ) {
+				$aria_label = esc_attr(
+					sprintf(
+						/* translators: Theme name. */
+						_x( 'Enable automatic updates for %s', 'theme', 'wp-autoupdates' ),
+						esc_html( $themes[ $stylesheet ]->get( 'Name' ) )
+					)
+				);
+				echo '<p class="theme-autoupdate-disabled">';
+				echo sprintf(
+					'<a href="%s" class="edit" aria-label="%s"><span class="dashicons dashicons-update" aria-hidden="true"></span> %s</a>',
+					wp_nonce_url( 'themes.php?action=autoupdate&amp;theme=' . urlencode( $stylesheet ) . '&amp;paged=' . $page . '&amp;theme_status=' . $theme_status, 'autoupdate-theme_' . $stylesheet ),
+					$aria_label,
+					__( 'Enable', 'wp-autoupdates' )
+				);
+				echo '</p>';
+			}
+		}
+	}
+}
+add_action( 'manage_themes_custom_column' , 'wp_autoupdates_add_themes_autoupdates_column_content', 10, 3 );
+
+
+/**
+ * Add plugins autoupdates bulk actions
```

Comment:
Themes, not plugins :)

Reply:
see, I knew I probably missed it somewhere :-).  Thanks!!